<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XScribbleClass</class>
 <widget class="QMainWindow" name="XScribbleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XScribble</string>
  </property>
  <widget class="ScribbleArea" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuDraw">
    <property name="title">
     <string>Draw</string>
    </property>
    <addaction name="actionDrawLine"/>
    <addaction name="actionDrawCircle"/>
    <addaction name="actionFreeHand"/>
    <addaction name="actionClear"/>
    <addaction name="actionClearall"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDraw"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDrawLine"/>
   <addaction name="actionDrawCircle"/>
   <addaction name="actionFreeHand"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="xscribble.qrc">
     <normaloff>:/XScribble/Resources/filesaveas.png</normaloff>:/XScribble/Resources/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionDrawLine">
   <property name="icon">
    <iconset resource="xscribble.qrc">
     <normaloff>:/XScribble/Resources/line_32.png</normaloff>:/XScribble/Resources/line_32.png</iconset>
   </property>
   <property name="text">
    <string>DrawLine</string>
   </property>
  </action>
  <action name="actionDrawCircle">
   <property name="icon">
    <iconset resource="xscribble.qrc">
     <normaloff>:/XScribble/Resources/radius.gif</normaloff>:/XScribble/Resources/radius.gif</iconset>
   </property>
   <property name="text">
    <string>DrawCircle</string>
   </property>
  </action>
  <action name="actionFreeHand">
   <property name="icon">
    <iconset resource="xscribble.qrc">
     <normaloff>:/XScribble/Resources/scribble1.jpg</normaloff>:/XScribble/Resources/scribble1.jpg</iconset>
   </property>
   <property name="text">
    <string>FreeHand</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionClearall">
   <property name="text">
    <string>Clearall</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionDrawStraightLine">
   <property name="text">
    <string>DrawStraightLine</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ScribbleArea</class>
   <extends>QWidget</extends>
   <header>scribblearea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="xscribble.qrc"/>
 </resources>
 <connections/>
</ui>
